test/e2e/predictor/test_triton.py

```diff
@@ -32,6 +32,7 @@ def test_triton():
         min_replicas=1,
         triton=V1beta1TritonSpec(
             storage_uri='gs://kfserving-examples/models/torchscript',
+            protocol_version="v2",
```

Comment:
- To maintain backwards compatibility we can default the protocol version to v2 if the runtime is set to triton. 
- if protocol version is specified while runtime is not, we can do the auto selection based on protocol and model format, e.g for tensorflow if v1 protocol is specified it assigns the tfserving runtime, when v2 is specified it assigns the triton runtime.

Reply:
@yuzisun updated!